src/test/scala/edu/ie3/simona/event/listener/RuntimeTestData.scala

```diff
@@ -0,0 +1,18 @@
+/*
+ * © 2024. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.simona.event.listener
+
+trait RuntimeTestData {
+
+  protected val startDateTimeString: String = "2011-01-01T00:00:00Z"
+  protected val endTick: Long = 3600
+  protected val duration: Long = 10805000
```

Comment:
A minor thing you can change if you like but I'm fine with keeping it as well. Duration can be misinterpreted and raises confusion why duration >> endTick. Maybe renaming it to something like simulationRuntimeDuration or similar can prevend this.

Reply:
Good point, added some explanatory ScalaDoc